src/MeetingsSDKAdapter.js

```diff
@@ -249,6 +256,26 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Returns an observable that emits the display data of a meeting control.
+   *
+   * @returns {Observable.<MeetingControlDisplay>}
+   * @memberof MeetingJSONAdapter
+   * @private
+   */
+  exitControl() {
+    return Observable.create((observer) => {
+      observer.next({
+        ID: EXIT_CONTROL,
+        icon: 'cancel',
+        tooltip: 'Leave meeting',
```

Comment:
Tooltip should just be "leave"
```suggestion
        tooltip: 'Leave',
```